src/com/spikes2212/genericsubsystems/drivetrains/commands/DriveTank.java

```diff
@@ -7,70 +7,79 @@
 import edu.wpi.first.wpilibj.command.Command;
 
 /**
- * This command moves a {@link TankDrivetrain} by giving each side a different speed.
- *
- * @author Unknown
+ * This command moves a {@link TankDrivetrain} using left and right speeds.
  */
```

Comment:
"this command ... using speeds **supplied to the left and the right sides independently**"